src/web/src/email_feedback.ecpp

```diff
@@ -259,17 +260,19 @@ UserInfo user;
                     "Error: client->sendto (address = '%s', subject = '%s', timeout = 1000) failed.",
                         AGENT_FTY_EMAIL_SENDMAIL_ONLY, "SENDMAIL");
         s_rm_rf (paths, &temp_dir);
-        http_die ("internal-error", "client->sendto () failed");
+        http_die ("internal-error", TRANSLATE_ME ("client->sendto () failed"));
     }
 
     zmsg_t *recieve = client->recv (zuuid_str_canonical (uuid) , _timeout);
     zuuid_destroy (&uuid);
     if (!recieve)
     {
-        std::string msg = "Error: client->recv (timeout = '" + std::to_string (_timeout) + "') returned NULL";
-        log_error ("%s", msg.c_str ());
+        char *msg = zsys_sprintf (TRANSLATE_ME ("Error: client-> recv (timeout = '%d') returned NULL", _timeout));
+        std::string msg_str (msg);
+        zstr_free (&msg);
+        log_error ("%s", msg_str.c_str ());
         s_rm_rf (paths, &temp_dir);
-        http_die ("internal-error", msg.c_str ());
+        http_die ("internal-error", msg_str.c_str ());
```

Comment:
Just curious: what does this trick achieve, compared to passing `msg` straight to `http_die()`? Better handling of releasing the memory during "die"? Or some character-type handling (unicode etc)?